### DISCUSSION TITLE
Legality of using this for research

### DISCUSSION THREAD

**Discussion by ChinJY** [zetavg/LLaMA-LoRA-Tuner#22] [Q&A]
I would like to fine-tune alpaca for a research paper, but I am wondering if it is legal to do so using this tuner. If it isn't, does anyone have suggestions for alternatives? Any advice would be greatly appreciated, thank you!

**Comment by zetavg** [comment]
Hi! Feel free to use or modify the code I wrote.

But for more advanced use cases, I would suggest using the Hugging Face transformers library directly rather than this UI tool for more control.

**Reply by ChinJY** [reply]
Thanks for the reply! Sorry for the late response by the way, I was not checking github.
